site/src/components/layout/rectangles.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: component
+title: Rectangles
+component: layout/rectangles.js
+namespace: layout
+
+example-code: |
+
+  // a very simple example component
+  function label(selection) {
+      selection.append('rect')
+          .layout('flex', 1);
+      selection.append('text')
+          .text(function(d) { return d.data; })
+          .attr({x: 20, y: 18});
+      selection.layout();
+  }
+
+  // the labels data
+  var data = [
+      { x: 100, y: 20, data: 'Hello'},
+      { x: 50, y: 100, data: 'World'},
+      { x: 150, y: 75, data: 'd3fc'},
+      { x: 250, y: 150, data: 'rocks'}
+  ];
+
+  // create an SVG container
+  var width = 400, height = 200;
+  var svg = d3.select('#layout')
+      .append('svg')
+      .attr('width', width)
+      .attr('height', height);
+
+  // create scales that span the width / height of the SVG
+  var xScale = d3.scale.linear()
+      .range([0, width]);
+  var yScale = d3.scale.linear()
+      .range([height, 0]);
+
+  // create the layout
+  var layout = fc.layout.rectangles()
+      .xScale(xScale)
+      .yScale(yScale)
+      .size([80, 30])
+      .position([function(d) { return d.x; }, function(d) { return d.y; }])
+      .component(label);
+
+  // bind the data and render
+  svg.datum(data)
+      .call(layout);
+---
+The rectangles layout component provides a mechanism for child components based on rectangular bounding boxes. It is typically used to render tooltips or labels on a chart.
+
```

Comment:
I'm not entirely sure what this first sentence means. I was expecting more of a verb after the words 'mechanism for' (i.e. "provides a mechanism for child components, based on rectangular bounding boxes, ... to do something").